### ISSUE TITLE
Stories: CSE Machine not available in the UI

### ISSUE THREAD

**Issue by martin-henz** [source-academy/frontend#2682]
The stories workspaces for Chapter 3 and 4 should show the CSE Machine.
<img width="1055" alt="Screenshot 2023-09-18 at 5 56 10 PM" src="https://github.com/source-academy/frontend/assets/30220768/5261bbee-b473-4c96-8917-71d0e617654e">

**Comment by RichDom2185** [comment]
Last time I recalled, CSE Machine gets confused due to the different code blocks sharing the same environment, hence I commented away the tab. Will look into this again.

**Comment by martin-henz** [comment]
Yes, would be great. Then I can make a story to introduce the tool. We have a story for each of our tools, so far.
